pkg/loop/internal/pb/gatewayconnector/gateway_common.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+option go_package = "github.com/smartcontractkit/chainlink-common/pkg/loop/internal/gatewayconnector";
+
+package loop;
+
+import "google/protobuf/empty.proto";
+
+message MessageBody {
```

Comment:
I don't like that the Message struct is duplicated as a proto. It was supposed to be an internal representation. Over the wire we send JSON-RPC-encoded messages. If @DeividasK adds his own version of Message, we would need to duplicate it too. 

Another option to consider is to send raw bytes in the gRPC service and decode in the Handler using our own JSON-RPC codec. WDYT? cc @cedric-cordenier 

Longer term, when we move to don2don framework for Gateways, we'll be passing raw bytes anyway.